packages/content/src/components/content-header/content-header.tsx

```diff
@@ -0,0 +1,88 @@
+          fill="#FAAD4D"
+        />
+      </symbol>
+      <mask id="bg-stars">
+        <use href="#bg-star" transform="translate(0 0)" />
+        <use href="#bg-star" transform="translate(18 0)" />
+        <use href="#bg-star" transform="translate(36 0)" />
+        <use href="#bg-star" transform="translate(54 0)" />
+        <use href="#bg-star" transform="translate(72 0)" />
+      </mask>
+      <mask id="stars">
+        <use href="#star" transform="translate(0 0)" />
+        <use href="#star" transform="translate(18 0)" />
+        <use href="#star" transform="translate(36 0)" />
+        <use href="#star" transform="translate(54 0)" />
+        <use href="#star" transform="translate(72 0)" />
+      </mask>
+      <rect x="0" y="0" width="100%" height="100%" fill="#D1D5DB" mask="url(#bg-stars)" />
+      <rect x="0" y="0" width={maskWidth} height="100%" fill="#FAAD4D" mask="url(#stars)" />
+    </svg>
+  );
+};
+
+const ContentHeader = (props: {
+  contentKind: ContentKind;
+  slug: string;
+  showStars: boolean;
+  showImage: boolean;
+}): JSX.Element => {
+  const { data, error } = useCourseGroupBySlugQuery({
+    variables: { Slug: props.slug }
+  });
+
+  if (error) {
+    console.log(error);
+  }
+
+  return (
+    // container
+    <div className="py-24 px-20">
+      <div className="px-1">
+        {/* header */}
+        <div className="text-5xl font-bold pb-2 font-primary">{data?.CourseGroupBySlug?.title}</div>
+        {/* description */}
+        <div className="text-lg text-slate-400 pb-2.5 font-primary">
+          {data?.CourseGroupBySlug?.description}
+        </div>
+        {/* stars */}
+        {props.showStars && (
+          <div className="flex pb-8">
+            {data?.CourseGroupBySlug?.rating && <Stars score={data?.CourseGroupBySlug?.rating} />}
+            {data?.CourseGroupBySlug?.rating && (
+              <div className="font-bold px-1">
+                {(data?.CourseGroupBySlug?.rating / 20).toFixed(1)}
+              </div>
+            )}
+            <div>({data?.CourseGroupBySlug?.ratingsCount} Reviews)</div>
```

Comment:
Know it'll be handled on the backend, but we should probably test for a value for `ratingsCount` and fallback to a `0` so we don't end up rendering something like ` Reviews`.

Reply:
Should we default to 0, or should we emit the reviews element entirely if there are no reviews?

Reply:
After pushing a fix, let me know if this is a suitable solution

Reply:
@AlexONeillTI That still assumes we'll always get back a `ratingsCount` if we get `ratings`; feels like the `ratingsCount` text should just be dependent on the `ratingsCount` var. Plus, if the `ratings` field is `0`, we'd end up 'hiding' this display entirely, which would mean it's essentially hiding bad (but valid) reviews. As to whether it should display `0 Reviews` or hide it entirely, that's more of a product/design decision.